package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rubynepal.github.io",
+  "version": "1.0.0",
+  "description": "Ruby Nepal Official Website",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack --mode development",
+    "watch": "webpack --mode development --watch",
+    "build": "webpack --mode production"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/madhusudhan1234/rubynepal.github.io.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/madhusudhan1234/rubynepal.github.io/issues"
```

Comment:
Better to change this link to `https://github.com/rubynepal/rubynepal.github.io/issues`

Reply:
okay dai, I will change 👍